modules/raft-client/src/main/java/org/apache/ignite/raft/client/service/RaftGroupService.java

```diff
@@ -49,7 +49,7 @@ public interface RaftGroupService {
     /**
      * Returns group id.
      */
-    String groupId();
+    ReplicationGroupId groupId();
```

Comment:
Is this OK to make this replacement if we don't fully own a RAFT code? Can you consult with @alievmirza?

Reply:
I guess it's OK after all, I'm just paranoid about raft packages

Reply:
What replacement are you talking about?

Reply:
String -> ReplicationGroupId

Reply:
Seems that this changes was introduced by Vlad today, I'm not aware of them